osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -127,6 +111,10 @@ void removeOldInfo()
                     Info?.FadeOut(250);
                     Info?.Expire();
                     Info = null;
+
+                    Background?.FadeOut(250);
+                    Background?.Expire();
+                    Background = null;
```

Comment:
Sure, can do. I can't grasp the overhead of using a "whole" container "just" to reduce code duplication

Reply:
What I mean is to have, using a tree structure:

```
Container
	Info
	Background
```

Then you just fade/expire the uppermost `Container` rather than both pieces individually.

Reply:
Yeah understood that, i only don't know how much overhead a container produces. If it's completely negligible, ofc it's better structurewise, if it has an impact i dont know how much it is and when it justifies it to reduce code duplication, i just cant measure it, therefor i will completely trust you on that and will do it